octavia_f5/controller/worker/flows/f5_flows_iseries.py

```diff
@@ -302,24 +302,25 @@ def make_ensure_vcmp_l2_flow(self) -> flow.Flow:
         get_existing_vlan = self.tasks.GetExistingVLAN()
         ensure_vlan = self.tasks.EnsureVLAN()
         ensure_vlan_interface = self.tasks.EnsureVLANInterface()
-        ensure_guest_vlan = self.tasks.EnsureGuestVLAN()
+        assign_vlan_to_guest = self.tasks.AssignVLANToGuest()
 
         ensure_vcmp_l2_flow = linear_flow.Flow('ensure-vcmp-l2-flow')
         ensure_vcmp_l2_flow.add(get_existing_vlan,
                                 ensure_vlan,
                                 ensure_vlan_interface,
-                                ensure_guest_vlan)
+                                assign_vlan_to_guest)
         return ensure_vcmp_l2_flow
 
     def make_remove_vcmp_l2_flow(self) -> flow.Flow:
         get_vcmp_guests = self.tasks.GetVCMPGuests()
-        remove_guest_vlan = self.tasks.RemoveGuestVLAN()
-        # no need to unassign the VLAN from the interface/LAG, because that is
-        # going to happen at VLAN deletioni anyway.
-        remove_vlan_if_not_owned_by_guest = self.tasks.RemoveVLANIfNotOwnedByGuest()
+        remove_vlan_from_guest = self.tasks.UnassignVLANFromGuest()
```

Comment:
The `remove_vlan_from_guest` is a bit confusing here. It could be interpreted as "removing (deleting) a vlan from the actual guest device. I know that this is here in context of the vcmp host flow but still. I would suggest renaming this to `unassign_vlan_from_guest`, it would then also match the task name.